packages/internal-test-env/index.ts

```diff
@@ -322,6 +322,8 @@ function validateLibVars(varsToValidate: LibraryVariables): object {
         type: "ESS Client Credentials",
         id: process.env.E2E_TEST_OWNER_CLIENT_ID,
         secret: process.env.E2E_TEST_OWNER_CLIENT_SECRET,
+        login: process.env.E2E_TEST_USER,
+        password: process.env.E2E_TEST_PASSWORD
```

Comment:
This is the only change to our library that gets distributed. The rest is e2e testing.